manifests/init.pp

```diff
@@ -29,8 +29,10 @@
   $tomcat = $gutterball::params::tomcat,
 ) inherits gutterball::params {
 
+  $_ = cache_data('gutterball_db_password', $dbpassword)
+  $__ = cache_data('gutterball_db_user', $dbuser)
+  $___ = cache_data('gutterball_tomcat', $tomcat)
```

Comment:
I'm not sure what this solves. The use case for cache_data is typically ```cache_data('my_password', random_password(16))```. See https://github.com/theforeman/puppet-foreman/blob/1daeb39c97a62d88a831c01004190052df1e4bdb/manifests/params.pp#L68-L70 for the more common implementation.